woodwork/tests/accessor/test_statistics.py

```diff
@@ -648,6 +654,45 @@ def __call__(self, update, progress, total, unit, time_elapsed):
     assert mock_callback.total_elapsed_time > 0
 
 
+def test_describe_dict_extra_stats(describe_df):
+    describe_df = describe_df.drop(columns=['boolean_col', 'natural_language_col', 'formatted_datetime_col',
+                                            'timedelta_col', 'latlong_col'])
+    describe_df['nullable_integer_col'] = describe_df['numeric_col']
+    describe_df['integer_col'] = describe_df['numeric_col'].fillna(0)
+    describe_df['small_range_col'] = describe_df['numeric_col'].fillna(0) // 10
+
+    ltypes = {
+        'category_col': 'Categorical',
+        'datetime_col': 'Datetime',
+        'numeric_col': 'Double',
+        'nullable_integer_col': 'IntegerNullable',
+        'integer_col': 'Integer',
+        'small_range_col': 'Integer',
+    }
+    describe_df.ww.init(index='index_col', logical_types=ltypes)
+    desc_dict = describe_df.ww.describe_dict(extra_stats=True)
```

Comment:
can we add a check that confirms that the extra stats are not present in a dictionary where extra stats=False?

Reply:
I updated `test_get_describe_dict` to confirm that the extra stats rows are not present.